frontend/packages/pipelines-plugin/src/components/taskruns/list-page/TaskRunsListPage.tsx

```diff
@@ -1,9 +1,18 @@
 import * as React from 'react';
+import * as _ from 'lodash';
 import Helmet from 'react-helmet';
 import { useTranslation } from 'react-i18next';
 import { ListPage } from '@console/internal/components/factory';
 import { getURLSearchParams } from '@console/internal/components/utils';
 import { referenceForModel } from '@console/internal/module/k8s';
+import {
+  COLUMN_MANAGEMENT_CONFIGMAP_KEY,
+  COLUMN_MANAGEMENT_LOCAL_STORAGE_KEY,
+  TableColumnsType,
+  useUserSettingsCompatibility,
+} from '@console/shared';
+import { TableColumn } from 'packages/console-dynamic-plugin-sdk/dist/core/lib/extensions';
+import { TaskRunKind } from 'packages/pipelines-plugin/src/types';
```

Comment:
Use a relative path if importing from the same package

```suggestion
import { TaskRunKind } from '../../../types/taskRun';
```